yabt/__init__.py

```diff
@@ -23,5 +23,5 @@
 
 
 __author__ = 'Itamar Ostricher'
-__version__ = '0.3.16'
+__version__ = '0.3.17'
```

Comment:
don't promote version manually as part of PR. use `make release` after merging into master.